patches/octane_run_we.js

```diff
@@ -0,0 +1,47 @@
+load('base.js');
+load('richards.js');
+load('deltablue.js');
+load('crypto.js');
+load('raytrace.js');
+load('earley-boyer.js');
+load('regexp.js');
+load('splay.js');
+load('navier-stokes.js');
+load('pdfjs.js');
+//load('mandreel.js');
```

Comment:
[Summary: yes, they're skipped because they're broken, but it makes it easier to see explicitly which are skipped.]